internal/codegen/golang/query.gotemplate

```diff
@@ -229,7 +229,7 @@ type textPreferrer struct {
 func (t textPreferrer) PreferredParamFormat() int16 { return pgtype.TextFormatCode }
 
 func (t textPreferrer) NewTypeValue() pgtype.Value {
-	return textPreferrer{pgtype.NewValue(t.ValueTranscoder).(pgtype.ValueTranscoder), t.typeName}
+	return textPreferrer{ValueTranscoder: pgtype.NewValue(t.ValueTranscoder).(pgtype.ValueTranscoder), typeName: t.typeName}
```

Comment:
I thought fieldalignment checked for the ordering of fields at the struct definition site. This looks like a lint to use named notation for struct fields instead of the positional notation.

Reply:
You're correct, `fieldalignment` only checks the struct definition site. But with the `-fix` flag it can also reorder the fields order. This breaks the instantiation of structs using positional notation. This PR adds the named notation to the few places that do not use it to prevent `fieldalignment -fix` from breaking the generated code.

Note that with its current definition, `textPreferrer` doesn't change or break anything. But `compositeField` does.